src/styles/index.scss

```
@@ -1,3 +1,90 @@
+@import "variables";
+@import "stars";
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  font-size: 12px;
+  line-height: 18px;
+  color: $main-accent;
+  font-family: $main-font;
+
+  .card-block {
+    width: 200px;
+    padding: 32px 16px 16px;
+    border: 1px solid #F3F3F3;
+    border-radius: 5px;
+    display: flex;
+    flex-direction: column;
+
+    .card-img {
+      width: 160px;
+      height: 134px;
+      margin-left: 3px;
+    }
```

it is incorrect usage of scss nesting. fix it everywhere
```suggestion
  .card {
    width: 200px;
    padding: 32px 16px 16px;
    border: 1px solid #F3F3F3;
    border-radius: 5px;
    display: flex;
    flex-direction: column;

    .&__img {
      width: 160px;
      height: 134px;
      margin-left: 3px;
    }
```